### DISCUSSION TITLE
Iterate through each element in a particular column of a dynamic table

### DISCUSSION THREAD

**Discussion by BrindaSateesh** [cypress-io/cypress#17618] [Questions and Help]
Hi all,

In my project, I have a dynamic table getting displayed based on a value selected from a radio button.  The radio button field "Doctor Name" field has different choices like "Frank", "Michael", "Josh", "Jessica".  When I select the "Frank" value, it displays a dynamic table with the list of appointments for "Frank",  The first column in the table is "Doctor Name".  When I select "Frank" from the radio button, I have to validate if all the appointments listed are for "Frank".  So I have to write coding in cypress to check if all the first column cell values are "Frank".  How can I achieve this?

**Comment by sandra-ouadghiri** [comment]
I mean, with what you explain, there's no way to reply. It depends on the DOM.
Use the Cypress doc, open your dev tool to look at the dom, try some code. Then share precise questions based on DOM + code element you provide.
